docker-compose-archv3.yml

```diff
@@ -0,0 +1,117 @@
+version: '3.7'
+
+## Usage
+## export COUCHDB_USER=admin && export COUCHDB_PASSWORD=admin123
+## docker-compose up -f docker-compose-archv3.yml
+## cntrl+c to exit
+## docker-compose down
+## docker volume rm medic-data medic-couchconfig
+
+## docker-compose up
+## https://localhost
+
+services:
+
+  couchdb:
+    container_name: medic-couchdb
```

Comment:
To avoid renaming in the future please use "cht" instead of "medic" for everything that self hosters will see.